flags.bzl

```diff
@@ -188,4 +188,13 @@ FLAGS = {
         The terminal width in columns. Configure this to override the default value based on what your CI system renders.
         """,
     ),
+    "test_summary": struct(
+        command = "test:ci",
```

Comment:
Why not

```suggestion
        command = "common:ci",
```

Reply:
It's a judgement call, I just think no engineer would expect to have a test-only flag applied to a non-test command. This one is labelled `terminal_output` https://bazel.build/reference/command-line-reference#build-flag--test_summary so we know it's safe and won't cause an analysis cache discard.